iOS/Flows/Conversation/Cells/TimeMachineCollectionViewLayout.swift

```diff
@@ -41,7 +41,6 @@ class TimeMachineCollectionViewLayout: UICollectionViewLayout {
     // MARK: - Data Source
     weak var dataSource: TimeMachineCollectionViewLayoutDataSource?
     weak var delegate: TimeMachineCollectionViewLayoutDelegate?
-    private var lastFrontmostIndexPath: [SectionIndex: IndexPath] = [:]
```

Comment:
I made the time machine layout more aggressive in how it reports the front most. It now always reports even if it's a duplicate.